toptal/htmlshell
# [TRA-1720] Cleanup HTML Shell

[TRA-1720](https://toptal-core.atlassian.net/browse/TRA-1720)

### Description

Cleaning up HTML Shell.

### How to test

- check staging (present on the ticket)

or

- pull the branch
- open `index.html` and check if the site keeps working as expected.

### Changes done

- optimized and resized images
- added `.editorconfig`
- formatted all `.html`, `.css` and `.js` files
- cleaned up messages on the header
- removed any files that were not being used in the project
- changed font size for better readability
- changed all github links to ours
- fixed `favicon` naming
- fixed logo size and aspect ratio
